node/directives/helper.ts

```diff
@@ -29,7 +29,12 @@ export const validateAdminToken = async (
       hasCurrentValidAdminToken = true
 
       if (authUser?.audience === 'admin') {
-        hasValidAdminToken = true
+        const hasAdminPermissions = await lm.getUserAdminPermissions(
```

Comment:
ideally we should log if this call to LM fails.

Reply:
(right now we are catching any exceptions inside `getUserAdminPermissions` and it will return false only, so we won't know if the token is not valid because it really isn't or because the call to LM failed)

Reply:
Also added logs for API and store token validations